tests/libcxx/CMakeLists.txt

```diff
@@ -20,7 +20,8 @@ foreach(testcase ${alltests})
 # the allocations are entirely optimized out by Clang in these tests and are excluded from Clang release builds -- Skip running these tests
     if ("${name}" MATCHES "cons_default_throws_bad_alloc.pass" OR "${name}" MATCHES "allocator_allocator.members_construct.pass")
         string(TOUPPER ${CMAKE_BUILD_TYPE} BUILD_TYPE_UPPER)
-        if (CMAKE_CXX_COMPILER_ID MATCHES Clang AND BUILD_TYPE_UPPER MATCHES REL)
+        if ((CMAKE_CXX_COMPILER_ID MATCHES Clang OR ENCLAVE_TESTS_CXX_COMPILER_ID MATCHES Clang)
```

Comment:
Consider consolidating the check into a single variable:
```cmake
if (ADD_WINDOWS_ENCLAVE_TESTS)
    # Load target LINUX_CXX_COMPILER_ID and LINUX_CXX_COMPILER_VERSION values
    set(ENCLAVE_CXX_COMPILER_ID ${LINUX_CXX_COMPILER_ID})
    set(ENCLAVE_CXX_COMPILER_VERSION ${LINUX_CXX_COMPILER_VERSION})
else()
    set(ENCLAVE_CXX_COMPILER_ID ${CMAKE_CXX_COMPILER_ID})
    set(ENCLAVE_CXX_COMPILER_VERSION ${CMAKE_CXX_COMPILER_VERSION})
endif()

...

if (ENCLAVE_CXX_COMPILER_ID MATCHES Clang)
...
```

Reply:
Update based on your suggestion.